app/sass/layouts/_cart.scss

```diff
@@ -0,0 +1,11 @@
+section {
+  @include outer-container;
```

Comment:
won’t this affect all `section` elements on the page? This selector seems to do a lot more than affect items in the “cart”